Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/WebView.cs

```diff
@@ -0,0 +1,38 @@
+﻿
+namespace Xamarin.Forms.PlatformConfiguration.AndroidSpecific
+{
+	using FormsElement = Forms.WebView;
+
+	public enum MixedContentHandling
+	{
+		AlwaysAllow = 0,
+		NeverAllow = 1,
+		CompatibilityMode = 2
+	}
+
+	public static class WebView
+    {
```

Comment:
can we double check code style on this file

Reply:
that last { is correct aligned when I look at it in my editors.

Reply:
it looks like somehow spaces snuck in instead of tabs here, we should be good now 😄